### ISSUE TITLE
[stm32f2][stm32f4] [doc] Simultaneous Dual DAC with DMA

### ISSUE THREAD

**Issue by vvirag** [libopencm3/libopencm3#528]
The documentation is generated from dac_common_all.c, and because the DAC API is common for all the stm32 architectures, each of them will include the documentation written there.

Among others, there is an example code and explanation on how the dual channel DAC supposed to be configured with DMA. However, this only works for the stm32_l1f013 boards, because for the stm32_f24 boards there are different API for the DMA handling.

In other words, we end up having an incompatible example code in the stm32_f24 code documentation.

Additionally, it fails to show how the data should be 1) constructed, 2) passed by for the dual channel (which is quite essential to make this setup work).

**Comment by vvirag** [comment]
We can keep the example for the DAC settings, and for the DMA part on the stm32_f24 I was thinking something like that:

```c
// ...
for (int i = 0; i < data_length; i++) {
	data[i]= data_for_dac1[i] | (data_for_dac2[i] << 16);
}
// ...
dma_set_memory_size(DMA1, DMA_STREAM5, DMA_SxCR_MSIZE_32BIT);
dma_set_peripheral_size(DMA1, DMA_STREAM5, DMA_SxCR_PSIZE_32BIT);
dma_set_transfer_mode(DMA1, DMA_STREAM5, DMA_SxCR_DIR_MEM_TO_PERIPHERAL);
dma_set_peripheral_address(DMA1, DMA_STREAM5, (uint32_t) &DAC_DHR12RD);
dma_set_memory_address(DMA1, DMA_STREAM5, (uint32_t) data);
dma_set_number_of_data(DMA1, DMA_STREAM5, data_size);
dma_channel_select(DMA1, DMA_STREAM5, DMA_SxCR_CHSEL_7);
// ...
```
